Toolset/palettes/standalone settings/revstandalonesettingsiosbehavior.livecodescript

```diff
@@ -379,12 +389,17 @@ function getProfiles
 end getProfiles
 
 on updateIosSettingState
-   set the enabled of group "iOS Settings" of me to (the hilite of button "Build for iOS" of me)
-   set the visible of button "iOSChecked" to (the hilite of button "Build for iOS" of me)
```

Comment:
This issue is not addressed yet, so make sure you put back those two lines (since they update the state of the "Green tick"